README.md

```diff
@@ -275,12 +274,27 @@ This waiting period allows time for community comment and review, and to provide
 If there is some urgent need for the package to be registered (for example, an upcoming talk featuring the package or other external deadline), the waiting period can be expedited by request made in the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) channel on the 
 public Julia Language Slack, or directly on the PR itself. This request should include a link to the registration PR and why an expedited merge is needed. In this case, a registry maintainer (see below) can manually review the package (often with stricter standards than the typical package faces, given the lack of bandwidth to review all packages thoroughly).
 
-#### Who can approve an early merge?
+### Who can approve an early merge?
 
 Any registry maintainer may merge a package registration PR early.
 Early merges should still be discussed on the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) slack channel before merging.
 Registry maintainers are discouraged from merging packages they directly *or* indirectly maintain as mentioned in [CONTRIBUTING.md](https://github.com/JuliaRegistries/General/blob/master/CONTRIBUTING.md#other-ways-to-help).
 
+### When is yanking a release appropriate?
+
+Releases that have already merged can be marked as "yanked" by adding `yanked = true` under the release entry in `Versions.toml`. This will mean 
+that Pkg will no longer consider them to exist and not resolve those versions, but that version cannot be re-registed in General. 
+
+The primary reason yanking exists is to make a release impossible to install.
+This is intended to be used for circumstances when having this release installed poses a risk to security or safety.
+It is not intended for avoiding people installing releases with bugs, for that use a patch release -- potentially after doing nothing more than revert. All patch releases prior to the most recent have known bugs by definition.
```

Comment:
While it is true that all patches prior have bugs, the goal of a patch release is to fix a bug. I agree that it would be weird to yank all prior variants of that release.

What should we do with a patch release that introduces a new bug? (And who determines criticalness?) The yank that has caused these recent discussions is was an atomic release (one that only changed one thing) and that introduced (from the kinds of usage the packages sees) a severe bug.

A new patch release reverting the change was made and the version was yanked. Modulo the judgement of the maintainer on severity that seems to be the exact purpose of yanking?

Reply:
> What should we do with a patch release that introduces a new bug?

Ask maintainers to make a new patch release.

> Modulo the judgement of the maintainer on severity that seems to be the exact purpose of yanking?

A severe package/usage bug is not generally a security vulnerability. A package merely not working correctly is different to that package causing security vulnerabilities down the road for users of that package.